packages/rn-tester/RNTester/NativeExampleViews/FlexibleSizeExampleView.mm

```diff
@@ -46,7 +46,7 @@ - (instancetype)initWithFrame:(CGRect)frame
 
     AppDelegate *appDelegate = (AppDelegate *)[[UIApplication sharedApplication] delegate];
 
-    _resizableRootView = [[RCTRootView alloc] initWithBridge:appDelegate.bridge
+    _resizableRootView = [[RCTRootView alloc] initWithBridge:appDelegate.rootViewFactory.bridge
```

Comment:
Question: Do we need to keep this backward compatible for user apps? So that they can reference `appDelegate.bridge`? I don't think it's okay to assume it's always there

Reply:
Edit: I used the `rootViewFactory` to init those views

![CleanShot 2024-01-18 at 16 38 36@2x](https://github.com/facebook/react-native/assets/52801365/04235fe4-b8e0-437d-a82a-d9388971f352)